src/library/OverflowContainer/OverflowContainerWithShadows.js

```diff
@@ -103,19 +143,33 @@ export default class OverflowContainerWithShadows extends Component<
   }
 
   render() {
-    const { hasShadowBottom, hasShadowTop } = this.state;
-    const isScrollable = hasShadowBottom || hasShadowTop;
+    const {
+      children,
+      hideScrollbars,
+      scrollX,
+      scrollY,
+      ...restProps
+    } = this.props;
+    const isScrollable = this.state !== {};
```

Comment:
This `isScrollable` check seems like a bit of an oversimplification.  Consider the following, which accounts for  the null state on first render and then checks for any truthy keys.

```js
const isScrollable = Boolean(
  this.state && Object.keys(this.state).filter((key) => this.state[key]).length
);
```